irohad/ametsuchi/impl/rocksdb_common.hpp

```diff
@@ -849,10 +986,11 @@ namespace iroha::ametsuchi {
       RocksDbCommon &common,
       expected::Result<rocksdb::Status, DbError> const &status) {
     std::optional<bool> value;
-    if constexpr (kOp == kDbOperation::kGet)
+    if constexpr (kOp == kDbOperation::kGet) {
```

Comment:
What is the reason to enable this function for kDbOperation other than kDbOperation::kGet?

Reply:
In all other cases loadValue() results to nullopt in compile time, strange.

Reply:
Yes, that's exactly how it was intended. To have single code with compile-time strict.